src/field/field3d.cxx

```diff
@@ -491,7 +491,25 @@ void Field3D::setBoundaryTo(const Field3D& f3d) {
 
   allocate(); // Make sure data allocated
 
-  /// Loop over boundary regions
+  if (isFci()) {
+    // Set yup/ydown using midpoint values from f3d
+    ASSERT1(f3d.hasParallelSlices());
+    ASSERT1(hasParallelSlices());
+
+    for (auto& region : fieldmesh->getBoundariesPar()) {
+      for (const auto& pnt : *region) {
+        // Interpolate midpoint value in f3d
+        const BoutReal val = pnt.interpolate_sheath_o1(f3d);
+        // Set the same boundary value in this field
+        pnt.dirichlet_o2(*this, val);
+      }
+    }
+    return;
```

Comment:
For hermes that might work, but should we not also copy the x-boundary conditions with FCI?
```suggestion
```